packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.test.ts

```diff
@@ -28,4 +30,94 @@ describe("getGradableGraph", () => {
 
         expect(result).toEqual(initialGraph);
     });
+
+    // (LEMS-2190): This test is to ensure that the new Mafs graph is reversing coordinates when the angle graph
+    // is reflexive in a clockwise direction in order to temporarily maintain the same angle scoring with the
+    // legacy graph. This logic (& the tests) should be moved to the scoring function after removing the legacy graph.
+    it("returns reversed coordinates if the angle graph is reflexive when not allowed", () => {
+        const state: InteractiveGraphState = {
```

Comment:
Thinking about the builder work that Ben/Nisha have done... in these three tests, the only bits that change are the `coords` and `allowReflexAngles` right? 

I wonder if it's worth building a InteractiveGraphState like this: 

```ts
const defaultState: InteractiveGraphState = {
    type: "angle",
    coords: [
        [-5, 0],
        [0, 0],
        [5, 5],
    ],
    hasBeenInteractedWith: true,
    range: [
        [-10, 10],
        [-10, 10],
    ],
    snapStep: [1, 1],
    showAngles: true,
    allowReflexAngles: false,
};
```

And then in the tests:

```ts
    const state = {
        ...defaultState,
        coords: [[], [], []],
        allowReflexAngles: true, // or false
    }
```

That would distill the data to only the bits that area essential to the test. 🤔